apps/frontend/src/libs/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,78 @@
+import { Icon } from "~/libs/components/components.js";
+import { useLocation } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+type BreadcrumbItem = {
+	link?: string;
+	name: string;
+};
+
+type Properties = {
+	items?: BreadcrumbItem[];
+};
+
+const FIRST_ITEM_INDEX = 0;
+const SINGLE_ITEM_OFFSET = 1;
+
+const formatBreadcrumbName = (name: string): string => {
+	return (
+		name
+			.replaceAll("-", " ")
+			// eslint-disable-next-line unicorn/prefer-string-replace-all
+			.replace(/\b\w/g, (char) => char.toUpperCase())
+	);
+};
+
+const Breadcrumbs = ({ items }: Properties): JSX.Element => {
+	const location = useLocation();
+	const pathnames = location.pathname.split("/").filter(Boolean);
```

Comment:
We don't need to calculate active item in Breadcrumbs. We will pass the predefined configuration from the page where it's used. 
```suggestion
```